vignettes/example-visualization.Rmd

```diff
@@ -0,0 +1,96 @@
+  root.dir = "../tests/data/TestProject/Code/"
+)
+```
+
+```{r, echo = FALSE, results = "hide", message = F}
+library(esqlabsR)
+projectConfiguration <- createDefaultProjectConfiguration()
+dataSheets <- "Laskin 1982.Group A"
+observedData <- loadObservedData(projectConfiguration = projectConfiguration, sheets = dataSheets)
+scenarioConfiguration <- ScenarioConfiguration$new(projectConfiguration)
+OutputPaths <- enum(list(
+  Aciclovir_PVB = "Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)"
+))
+simulatedScenarios <- runScenarios(
+  scenarioNames = c("TestScenario"),
+  scenarioConfiguration = scenarioConfiguration,
+  customParams = NULL, saveSimulationsToPKML = FALSE
+)
+```
+
+### Workflow
+
+Plotting the simulation results is an important part of model diagnostics and quality control. Simulated modeling scenarios can be passed to plotting functions from the `{ospsuite}` package to create uniformly-looking plots.
+
+`DataCombined` is a class used to store matching observed and simulated data. Initialize a new class instance and populate it with data with the following code:
+
+```{r datacombined}
+dataCombined <- DataCombined$new()
+dataCombined$addDataSets(observedData, names = "Observed", groups = "Aciclovir")
+dataCombined$addSimulationResults(simulatedScenarios$TestScenario$results, names = "Simulated", groups = "Aciclovir")
+dataCombined$toDataFrame()
+```
+
+The simulation results are stored in a list returned by the `runScenarios()` function. Plotting and visualization is performed by storing these results along with matching observed data in an instance of the `DataCombined` class and passing it to plotting functions.
+
+Plotting functions in the `{ospsuite}` package are wrappers around `{tlf}` plotting functions that provide default plot configuration options. All of them accept instances of `DataCombined` class as the data source. 
+
+Time profile plots visualize the pharmacokinetics of the drug in question and help assess if the observed data (shown with scatter plots) match the simulated data (shown with line plots).
+
+```{r plot time profile, fig.width=7, fig.height=4}
+plotIndividualTimeProfile(dataCombined)
+```
+
+Observed vs simulated plots show if simulated time points and observed time points follow a linear trend.
+
+```{r plot obs vs pred, fig.width=7, fig.height=4}
+plotObservedVsSimulated(dataCombined)
+```
+
+Residual plots show if there is a systematic bias in how the simulation represents values either in high-concentration or low-concentration regions, or, alternatively, in early or late time periods.
+
+```{r residuals vs simulated, fig.width=7, fig.height=4}
+plotResidualsVsSimulated(dataCombined)
+```
+
+```{r residuals vs time, fig.width=7, fig.height=4}
+plotResidualsVsTime(dataCombined)
+```
+
+The resulting plots (returned by plotting functions) are instances of `ggplot` class. They can be modified and saved to files with `{ggplot2}` functions:
+
+```{r save plots}
```

Comment:
```suggestion
```{r save-plots}
```

cf. https://github.com/Open-Systems-Pharmacology/Suite/blob/develop/CODING_STANDARDS_R.md#rmarkdown

Reply:
Please change all chunk labels accordingly.

Reply:
This should be resolved only when all chunk labels have been renamed to follow our coding conventions.